### PR TITLE
Kt/issue 1900

### PR DESCRIPTION
**Pull request by ktomala**
Fix for issue #1900. Please review and merge.

### CODE REVIEW

**Comment by maaktweluit** [comment]
Could you `rebase -i` the branch `mwu/bb-unit-tests` out of this PR?

This can go directly from and to develop since the files you touched are not changed in `mwu/bb-unit-tests`

As discussed I will run some tests on windows and mac, then review the rest of the PR.

File: scripts/test-daemon-start.ps1 [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 "Ensure hyperg is started"
 
 $HyperdriveProcess = Get-Process hyperg -ErrorAction SilentlyContinue
 
-if (!$HyperdriveProcess)
-{
-  "Starting..."
-  $HyperdriveProcess = Start-Process C:\BuildResources\hyperg\hyperg.exe -WorkingDirectory C:\Users\buildbot-worker -PassThru
+if ($HyperdriveProcess) {
+  Stop-Process -Id $HyperdriveProcess
```
Comment (maaktweluit) [inline scripts/test-daemon-start.ps1]: missing `.Id` after `$HyperdriveProcess`

**Review by maaktweluit — CHANGES_REQUESTED** [review]

**Comment by ktomala** [comment]
Branch was rebased.

**Comment by maaktweluit** [comment]
Thank you for the rebase!

Ran tests on mac as well, these went flawless.
Also see `buildbot/control_test` is green so looks good :)

Only the 1 windows comment left on the PR.

**Review by maaktweluit — APPROVED** [review]

**Comment by ktomala** [comment]
@badb Could we merge this PR?

**Pull request merged**